src/home/index.html

```diff
@@ -52,5 +53,173 @@ <h1 class="heading">I am Mwafrika Glad to see you!</h1>
         </ul>
       </div>
     </section>
+    <section class="works">

```

Comment:
#### OPTIONAL 

As the previous reviewer missed this issue, I'll mark this as a suggestion rather than a requirement so implement it if your wish 😁 

Your margins are having some issues in the works section that is causing your cards to not be centered 😥 

![Captura](https://user-images.githubusercontent.com/15807343/148632795-be3158d0-97fa-406d-a813-52fc7540d6a0.PNG)